### DISCUSSION TITLE
Guided Tour

### DISCUSSION THREAD

**Discussion by datntfe** [glideapps/glide-data-grid#818] [General]
Is there any way to do Guided Tour in this?

**Comment by jassmith** [comment]
Please explain more what you want.

**Comment by datntfe** [comment]
It like this https://elrumordelaluz.github.io/reactour/

But I can't selector to inside glide-data-grid

**Reply by aleemstreak** [reply]
i think its because its Canvas?

**Comment by jassmith** [comment]
In general its harder to use react-tour with this but not impossible. The DataEditorRef has a `getBounds` call which can be used to position an invisible DOM object over top of any cell you want. You can then create those elements for the duration of the tour then remove them when the tour completes.
